### ISSUE TITLE
Reduce build time

### ISSUE THREAD

**Issue by glennrfisher** [watson-developer-cloud/ios-sdk#398]
The SDK is very slow to build with Carthage. Part of the problem is that it requires all dependencies and schemes to be built. It's possible that we can zip up the Carthage build folder and include it with a release (e.g. `WatsonDeveloperCloud.framework.zip`). If so, then users would only have to download and extract the zip file instead of building each framework.

**Comment by glennrfisher** [comment]
I briefly tried this with an earlier release. I used the `carthage archive` command to create _separate_ zip files for each scheme. Unfortunately, testing showed that Carthage would only download and extract the alphabetically-first file of the release. It would be worth trying this again, but zipping together the entire Carthage directory (or otherwise matching the result of the `carthage archive` command, but including all frameworks in the zip file).

**Comment by glennrfisher** [comment]
If Carthage doesn't support downloading multiple frameworks from the same release, perhaps we could submit a pull request to handle this situation.

An alternative solution would be to separate this repository into a branch or repo for each service. This would allow people using Carthage to specify only the frameworks that they need for their project.

**Comment by alanscarpa** [comment]
How about cocoapods?

**Comment by glennrfisher** [comment]
We've made some great improvements with our newest releases: we figured out how to use the `carthage archive` command to build a .zip file with binaries of the frameworks.

When you run `carthage update` with one of the newer releases, it still builds our dependencies (Alamofire, Freddy, Starscream) but downloads the Watson frameworks instead of building them.

We found that uploading binaries reduces the build time from about 30 minutes to 5 minutes. It's still longer than we would like but it's a great step forward.

@alanscarpa: We've received a few requests for CocoaPods support. We have an open issue (#127) and would really like to get to it. Unfortunately, we have some other work that takes priority, but we'll get there!
